scene/gui/color_picker.h

```diff
@@ -93,14 +199,19 @@ class ColorPicker : public BoxContainer {
 	HSeparator *preset_separator = memnew(HSeparator);
 	Button *btn_add_preset = memnew(Button);
 	Button *btn_pick = memnew(Button);
-	CheckButton *btn_hsv = memnew(CheckButton);
-	CheckButton *btn_raw = memnew(CheckButton);
-	HSlider *scroll[4];
+
+	OptionButton *mode_option_button = memnew(OptionButton);
+
+	HSlider *scrolls[4];
 	SpinBox *values[4];
 	Label *labels[4];
 	Button *text_type = memnew(Button);
 	LineEdit *c_text = memnew(LineEdit);
 
+	HSlider *alpha_scroll;
+	SpinBox *alpha_value;
+	Label *alpha_label;
```

Comment:
```suggestion
	HSlider *alpha_scroll = nullptr;
	SpinBox *alpha_value = nullptr;
	Label *alpha_label = nullptr;
```
All pointers should be initialized to null. Not sure about pointer arrays.